src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,2122 @@
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove SpeechDetected Failed with error " + error);
+                }
+
+                _speechDetected -= value;
+                _beginSpeechDetectedDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Event to be invoked when the default language changes.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        /// <pre>
+        /// The state must be initialized.
+        /// </pre>
+        public static event EventHandler<CurrentLanguageChangedEventArgs> CurrentLanguageChanged
+        {
+            add
+            {
+                _languageDelegate = (IntPtr previousLanguage, IntPtr currentLanguage, IntPtr userData) =>
+            {
+                string previousLanguageString = Marshal.PtrToStringAnsi(previousLanguage);
+                string currentLanguageString = Marshal.PtrToStringAnsi(currentLanguage);
+                CurrentLanguageChangedEventArgs args = new CurrentLanguageChangedEventArgs(previousLanguageString, currentLanguageString);
+                _currentLanguageChanged?.Invoke(null, args);
+            };
+                ErrorCode error = VcMgrSetCurrentLanguageChangedCb(_languageDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add CurrentLanguageChanged Failed with error " + error);
+                }
+
+                else
+                {
+                    _currentLanguageChanged += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetCurrentLanguageChangedCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove CurrentLanguageChanged Failed with error " + error);
+                }
+
+                _currentLanguageChanged -= value;
+                _languageDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Gets the current error message.
+        /// This function should be called during as ErrorOccurred Event.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>
+        /// http://tizen.org/privilege/voicecontrol.manager
+        /// </privilege>
+        /// <privlevel>
+        /// public
+        /// </privlevel>
+        /// <feature>
+        /// http://tizen.org/feature/speech.control
+        /// http://tizen.org/feature/microphone
+        /// </feature>
+        public static string ErrorMessage
```

Comment:
Why need it?
Error message should delivered by `ErrorOccurred` event

Reply:
will VcMgrGetErrorMessage(out msg) function move in ErrorOccurred function.